### PR TITLE
doc: otlp specific exporter and remove unnecessary repos from pom

### PR DESCRIPTION
**Pull request by cmergenthaler**
Adds otlp specific exporter to Readme in order to have proper setup. See #155

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
Thanks for the PR! 

The README.adoc file gets generated from this file (https://github.com/spring-projects-experimental/spring-cloud-sleuth-otel/blob/main/docs/src/main/asciidoc/README.adoc). Can you update your PR to change that file.

Then you can run `./mvnw clean install -Pdocs -pl docs` and the files will be regenerated.

Can you also consider updating the docs in the how to section https://github.com/spring-projects-experimental/spring-cloud-sleuth-otel/blob/main/docs/src/main/asciidoc/howto.adoc (final version here https://spring-projects-experimental.github.io/spring-cloud-sleuth-otel/docs/1.1.1/reference/html/howto.html#howto)

**Comment by cmergenthaler** [comment]
@marcingrzejszczak Thanks for the reply! Should have read the docs before changing them.
I have changed the correct files now

File: README.adoc [hunk 1]
```diff
@@ -68,12 +68,12 @@ Add Spring Cloud Sleuth to the classpath of a Spring Boot application (together
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-sleuth-otel-autoconfigure</artifactId>
-        </dependency>
+        </dependency>`
```
Comment (marcingrzejszczak) [inline README.adoc:71]: What is this?

Comment (cmergenthaler) [reply README.adoc]: good catch, thanks. removed it

**Pull request merged**